wire-ios-sync-engine/Source/UserSession/ZMClientRegistrationStatus.swift

```diff
@@ -481,7 +481,7 @@ public class ZMClientRegistrationStatus: NSObject, ClientRegistrationDelegate {
     }
 
     @objc(didRegisterProteusClient:)
-    public func didRegisterProteusClient(_ client: UserClient) {
+    public func didRegisterProteusClient(_ client: UserClient) { // TODO: relevant for import?
```

Comment:
since we don't invalidate the client, we should not have to use both methods here, technically we keep the same client and backend does not know about it